design-guidelines/form-ux.md

```diff
@@ -0,0 +1,189 @@
+# Form Design Guidelines
+
+A form may be used to collect data that is required to complete a task.
+
+## Form contexts
+
+The context in which a form is rendered depends on the complexity of the form and how critical the form's purpose is to the rest of the app.
+
+### Full page
+
+A form may be rendered on it's own page helps users to focus on completing their task with minimal distractions in the UI.
+
+Render a form on it's own page if the form:
+
+- is complex and has many fields
+- will be used to complete an action that is very important
+
+### Section of a page
+
+A form may be rendered on a page alongside additional content to avoid taking users out of their current flow.
+
+Render a form alongside other content if:
+
+- the user may need to reference page content to help them fill out the form correctly
+- the form is not so complex that it deserves it's own page
+- it would be too disruptive to take a user away from the current page
+- the page has content that is related to the form
```

Comment:
any advice about URLs? Would there be any decision making around rendering a form in a section as associated with a particular route?

Reply:
Good point. Yes, I think we'd want it to have it's own route.